server/routes/meetup.route.js

```diff
@@ -33,6 +33,22 @@ router.get('/:id', function (req, res, next) {
   })
 });
 
+router.get('/sequence/:id', function (req, res, next) {
+  const id = req.params.id;
+
+  meetupService.getMeetupBySequenceNo(id, function (err, meetup) {
```

Comment:
Not sure if using callbacks is a good approach. Can we shift to either promises or async/await ?

Reply:
good idea. @TalhaKhatri lets move this code to promises for now. Im not sure which node version is running on the server